packit_service/service/app.py

```diff
@@ -62,46 +62,98 @@ def get_flask_application():
         handler.setFormatter(RFC5424Formatter(msgid=project))
         logger.addHandler(handler)
 
-    logger.info(
-        f"server name = {service_config.server_name}, all HTTP requests need to use this URL!",
-    )
-
     package_versions = [
         ("Flask", flask_version),
         ("Flask RestX", restx_version),
+        ("FastAPI", fastapi_version),
         ("Packit Service", ps_version),
     ]
     log_package_versions(package_versions)
 
+    return logger
+
+
+def get_flask_application():
+    app = Flask(__name__)
+    app.register_blueprint(blueprint)
+    service_config = ServiceConfig.get_service_config()
+    # https://flask.palletsprojects.com/en/1.1.x/config/#SERVER_NAME
+    # also needs to contain port if it's not 443
+    # TODO local deployment fails without uncommenting
+    # app.config["SERVER_NAME"] = service_config.server_name
+    app.config["PREFERRED_URL_SCHEME"] = "https"
+    if getenv("DEPLOYMENT") in ("dev", "stg"):
+        app.config["DEBUG"] = True
+
+    app.logger.setLevel(logging.DEBUG)
+
+    logger.info(
+        f"server name = {service_config.server_name}, all HTTP requests need to use this URL!",
+    )
     # no need to thank me, just buy me a beer
     logger.debug(f"URL map = {app.url_map}")
     return app
 
 
-packit_as_a_service = Proxy(get_flask_application)
+logger = setup_logging_and_sentry()
 
-CORS(packit_as_a_service)
+flask_app = get_flask_application()
 
-INLINE = [
-    "'unsafe-inline'",
-    "'self'",
-]
+CORS(flask_app)
+INLINE = ["'unsafe-inline'", "'self'"]
 Talisman(
-    packit_as_a_service,
-    # https://github.com/wntrblm/flask-talisman#options
-    # https://infosec.mozilla.org/guidelines/web_security#implementation-notes
+    flask_app,
```

Comment:
```suggestion
    # https://github.com/wntrblm/flask-talisman#options
    # https://infosec.mozilla.org/guidelines/web_security#implementation-notes
    flask_app,
```

Should we keep this links? I think they are still relevant.

Reply:
yes, I firstly removed this because I thought it is not necessary anymore with setting it on FastAPI level, but tests revealed this is still needed, good catch!